app/helpers/check_answers_helper.rb

```diff
@@ -1,154 +1,47 @@
-      next link_to_parent_page(question, "rooms_number") if question.eql?("accommodation_cost")
-      next link_to_parent_page(question, "offer_space_type") if question.eql?("space_cost")
-
-      value = concat_answer(session[question], question)
-
+  def sections
+    {
+      1 => "accommodation",
+      2 => "transport",
+      3 => "space",
+      4 => "staff",
+      5 => "care",
+      6 => "services",
+      7 => "other_support",
+      8 => "location",
+      9 => "business_details",
+      10 => "contact_details",
+    }.sort.to_h.values
+  end
+
+  def question_items(question)
+    if session[question].is_a?(Array)
+      [{
+        field: I18n.t("coronavirus_form.questions.#{question}.title"),
+        value: render("coronavirus_form/check_your_answers/helpers/list", items: session[question]),
+      }]
+    else
       [{
-        field: t("coronavirus_form.questions.#{question}.title"),
-        value: sanitize(value),
-        edit: {
-          href: "#{question.dasherize}?change-answer",
-        },
+        field: I18n.t("coronavirus_form.questions.#{question}.title"),
+        value: session[question],
       }]
     end
-
-    items.compact.flatten
-  end
-
-  def skip_question?(question)
-    question.in?(SKIPPABLE_QUESTIONS) && session[question].blank?
-  end
-
-  def transport_type
-    [{
-      field: t("coronavirus_form.questions.transport_type.title"),
-      value: sanitize(transport_type_info),
-      edit: {
-        href: "transport-type?change-answer",
-      },
-    }]
-  end
-
-  def transport_type_info
-    answer = []
-    answer << Array(session[:transport_type]).flatten.to_sentence
-    answer << session[:transport_description]
-
-    answer.join("<br>")
   end
 
-  def offer_care_qualifications
-    [{
-      field: t("coronavirus_form.questions.offer_care_qualifications.offer_care_type.title"),
-      value: sanitize(Array(session[:offer_care_type]).flatten.to_sentence),
-      edit: {
-        href: "offer-care-qualifications?change-answer",
+  def accommodation_items
+    [
+      {
+        field: I18n.t("coronavirus_form.check_your_answers.sections.accommodation.type"),
+        value: session[:rooms_number],
       },
-    },
-     {
-       field: t("coronavirus_form.questions.offer_care_qualifications.care_qualifications.title"),
-       value: sanitize(Array(session[:offer_care_qualifications]).flatten.to_sentence),
-       edit: {
-         href: "offer-care-qualifications?change-answer",
-       },
-     }]
-  end
-
-  def link_to_parent_page(question, parent_question)
-    [{
-      field: t("coronavirus_form.questions.#{question}.title"),
-      value: sanitize(concat_answer(session[question.to_sym], question)),
-      edit: {
-        href: "#{parent_question.dasherize}?change-answer",
+      {
+        field: I18n.t("coronavirus_form.check_your_answers.sections.accommodation.description"),
+        value: session[:accommodation_description],
```

Comment:
The edit links are no longer being added here. Can you add a note to the commit message explaining why they are no longer being generated here, and what the plan is for them?

Reply:
Done - comment on appropriate commit.  Thanks 👍